lib/puppet/file_serving/mount/tasks.rb

```diff
@@ -0,0 +1,21 @@
+require 'puppet/file_serving/mount'
+
+class Puppet::FileServing::Mount::Tasks < Puppet::FileServing::Mount
+  def find(path, request)
+    raise _("No task specified") if path.to_s.empty?
+    module_name, task_path = path.split("/", 2)
+    return nil unless mod = request.environment.module(module_name)
```

Comment:
Can the task_path contain special characters? If so, do we need to decode the `task_path` from a URI to a file path, or has the decoding already occurred by the time we're called? Could you add a test that we can retrieve a task_path with special characters?